benchmark/kvbench/main.py

```
@@ -58,6 +64,136 @@ def load_matrix(matrix_file) -> np.ndarray:
+                sections[current_section].append(line.split())
+            else:
+                # No section header yet -> legacy matrix file
+                sections.setdefault("rdma", []).append(line.split())
+
+    result: Dict[str, Any] = {"rdma": None, "read": None, "write": None}
+
+    if "rdma" in sections:
+        matrix = [[parse_size(x) for x in row] for row in sections["rdma"]]
+        result["rdma"] = np.array(matrix)
+
+    if "read" in sections and sections["read"]:
```

Ruff (0.15.14)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)